dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScopeTest.kt

```diff
@@ -362,6 +363,7 @@ internal class RumViewManagerScopeTest {
         testedScope = RumViewManagerScope(
             parentScope = mockParentScope,
             backgroundTrackingEnabled = false,
+            trackFrustrations = true,
```

Comment:
here and below we need to inject forged value instead. `backgroundTrackingEnabled` is set to the hardcoded value just because the test requires that value.